### ISSUE TITLE
Custom PYTHONPATH not picked up when importing UDFs into standalone add-in

### ISSUE THREAD

**Issue by abielr** [xlwings/xlwings#2209]
Windows 10, xlwings 0.29.1, Office 2016, Python 3.9.13

I have made a standalone Excel XLAM add-in that refers to UDFs contained in a Python file that will be stored in a shared drive, so it will not be in the same directory as the add-in nor will it be in the Python `/site-packages` directory. Per the instructions in the `Importing UDFs` section of the [Custom Add-in](https://docs.xlwings.org/en/latest/customaddin.html) documentation I run the `ImportPythonUDFsToAddin` macro. Before doing so I set the `UDF Modules` property in the `myaddin.conf` tab to the name of the file excluding the `.py` extension, and and I set `PYTHONPATH` equal to the directory it is contained in. However, when then running `ImportPythonUDFsToAddin` I get a `ModuleNotFoundError`. I have inserted a `print(sys.path)` statement into the xlwings `import_udfs` function so I can see what path it is operating with at the time it tries to import my module. The path does not contain the additional path I added to the `PYTHONPATH` variable in `myaddin.conf`. I can see in the console it is trying to import the correct module name, so I know it is picking up at least some of the variables in `myaddin.conf`.

**Comment by fzumstein** [comment]
Are you running `ImportPythonUDFsToAddin` from the xlwings module of your custom add-in or could you be running it from the xlwings add-in?

**Comment by abielr** [comment]
I realized this was coming from a modification I had made and then forgotten about in the `XLPyCommand` subroutine in which I had taken `ActiveWorkbook.FullName` out of the `PYTHONPATH` variable, thereby changing the number of elements at the start of of `PYTHONPATH`, which later causes a problem in `xlwings.utils.prepare_sys_path` since it expects that the first six paths are reserved for specific terms. 

I had removed `ActiveWorkbook.FullName` from `PYTHONPATH` based on the this discussion in [#787](https://github.com/xlwings/xlwings/issues/787) regarding how to avoid having a separate COM server launch for each workbook. I have further modified my approach to replace `ActiveWorkbook.FullName` with an empty string in the `PYTHONPATH` construction, which seems to do the job, although I'm not sure if this will later have unintended consequences or if there alternative recommended ways to accomplish the same thing.
